packages/client/components/NewMeetingSettingsRetrospectiveSettings.tsx

```diff
@@ -9,12 +9,18 @@ import {PortalStatus} from '../hooks/usePortal'
 import NewMeetingDropdown from './NewMeetingDropdown'
 import NewMeetingSettingsToggleAnonymity from './NewMeetingSettingsToggleAnonymity'
 import NewMeetingSettingsToggleCheckIn from './NewMeetingSettingsToggleCheckIn'
+import NewMeetingSettingsToggleTeamHealth from './NewMeetingSettingsToggleTeamHealth'
 
 const NewMeetingSettingsToggleCheckInMenuEntry = styled(NewMeetingSettingsToggleCheckIn)({
   background: 'none',
   borderRadius: 0
 })
 
+const NewMeetingSettingsToggleTeamHealthMenuEntry = styled(NewMeetingSettingsToggleTeamHealth)({
```

Comment:
You're probably already aware of this, but the meeting setting carousel will be deprecated for the new activity view at some point. But I imagine you'll build this feature before the activity view is ready for general release, so I think it's the right move to add it here.